bittide-instances/bin/Shake.hs

```diff
@@ -166,11 +166,10 @@ main = do
           -- Design should meet timing post routing. Note that this is not a
           -- requirement after synthesis as many of the optimizations only follow
           -- after.
-          report <- liftIO (ParseTimingSummary.parseFile postRouteTimingSummaryPath)
-          case ParseTimingSummary.meetsTiming report of
-            Nothing -> pure ()
-            Just wns -> error [I.i|
-              Design did not meet timing. Negative WNS detected: #{wns}. Check out
+          liftIO $ unlessM
+            (meetsTiming postRouteTimingSummaryPath)
+            (error [I.i|
+              Design did not meet timing. Negative WNS detected. Check out
```

Comment:
Since you don't do any WNS detection anymore, I think you should rephrase this to just say that timing constraints were not met.

Reply:
Done!